haiku-apps/projectconceptor/projectconceptor-1.0.0.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="A Haiku diagramming application like Microsoft Visio"
+DESCRIPTION="
+	ProjectConceptor is a modular application for Haiku and BeOS designed to \
+	view, edit and process information which can be represented by a graph \
+	structure.
+	"
```

Comment:
@owenca, could you change DESCRIPTION to:
```
DESCRIPTION="ProjectConceptor is a modular application for Haiku and BeOS \
designed to view, edit and process information which can be represented by a \graph structure."
```

Yes, this is our fault. The [example with the build recipe for gawk](https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#a-simple-example-using-an-outdated-style-for-description) was using an outdated style for `DESCRIPTION`, so I added an [example with the build recipe for which](https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#a-simple-example-using-the-recommended-style-for-description) which uses the recommended style.

(I'll check which are the mostly used styles for `COPYRIGHT` and `LICENSE`, so I might ask you to change these too.)
Thanks, and sorry for making these picky requests...